src/ch.usi.inf.nodeprof/js/analysis/api/minitests.invoke.js.expected

```diff
@@ -4,18 +4,8 @@
 [i] string level
 symbolic location ':program': (src/ch.usi.inf.nodeprof.test/js/minitests/invoke.js:{{:program}})
 symbolic location 'module': (src/ch.usi.inf.nodeprof.test/js/minitests/invoke.js:{{module}})
-this is global: true
-[Object: null prototype] {
-  iidToLocation: undefined,
-  iidToCode: undefined,
-  iidToSourceObject: undefined,
-  nativeLog: undefined,
-  valueOf: undefined,
-  onReady: undefined,
-  registerCallback: undefined,
-  instrumentationSwitch: undefined,
-  getConfig: undefined
-}
+this is global: false
+undefined
```

Comment:
@Haiyang-Sun can you elaborate why these semantics would be wrong?

Reply:
Sorry, I managed to confuse `this` and the callback argument `dis` in that test. Fixed in 33df011